src/wallet/db.cpp

```diff
@@ -167,8 +167,8 @@ bool CDB::Recover(const std::string& filename, void *callbackDataIn, bool (*reco
     // Call Salvage with fAggressive=true to
     // get as much data as possible.
     // Rewrite salvaged data to fresh wallet file
-    // Set -rescan so any missing transactions will be
-    // found.
+    // call rescanblockchain (RPC) so any missing
```

Comment:
It's confusing. But CDB::Recover only gets calls by setting `-salvagewallet` which does set `-rescan`. Maybe I should remove that comment part... I don't what's best because `-rescan` is currently mentioned there.